### ISSUE TITLE
Battlerite Support

### ISSUE THREAD

**Issue by Spazoman** [antonpup/Aurora#348]
Battlerite has lighting effects on razer keyboards, but would be nice to have it on my corsair keyboard

**Comment by antonpup** [comment]
Have you tried using Razer Wrapper with it?

**Comment by simon-wh** [comment]
Any updates on this?

**Comment by Spazoman** [comment]
Hi, sorry I've not been able to get back to this post until now. Yes, I have tried installing the razed wrapper, and it didn't work. However I'm not sure if I did everything correctly or if I was missing something. I've actually tried installing the wrappers on other games as well, and with some games it kind of worked, but mostly it had no effect

**Comment by Spazoman** [comment]
I should also add that the other games I tried to get working did in fact have lightning effects, and I wasn't just throwing it into a random game expecting my keyboard to flash colors.

**Comment by simon-wh** [comment]
@Spazoman Are you applying it to the game directory that contains the executable for the game?

**Comment by Spazoman** [comment]
Yes that is where I am putting the file. I've tried both 64bit and 32bit just in case, but still no results. I'm not at my computer at the moment but I can take another look at what I tried to do when I can.

**Comment by Dagda52** [comment]
Yes indeed, now that battlerite is f2p i wanted to test it and there is no signal of dlls being even called since there are no logs for example overwatch log this https://pastebin.com/XL5uSanC

But battlerite does nothing, i tried to install the wrapper on any folder possible of the game but it wont load, only desktop lighting is visible
